IBAnimatable/MaskDesignable.swift

```diff
@@ -31,6 +31,7 @@ public protocol MaskDesignable {
    ```
    */
   var maskType: MaskType { get set }
+  var previousMaskType: MaskType { get set }
```

Comment:
If we still need it. It will be better if we have some documentation comments here to explain what is the property for. e.g. 
```
/// `previousMaskType` is a property to cache the previous mask type. If it is not `.none`, then we will remove `layer.mask` before re-adding it.
```
Feel free to write a better comment for that 😉